packages/frontend/src/system-integration/notifications.ts

```diff
@@ -76,19 +110,62 @@ async function showNotification(
       chatId,
       messageId,
       accountId,
+      isWebxdcInfo,
     })
   } else {
     try {
-      const notificationInfo =
-        await BackendRemote.rpc.getMessageNotificationInfo(accountId, messageId)
-      const { chatName, summaryPrefix, summaryText } = notificationInfo
+      let chatName = ''
```

Comment:
`chatName` is empty for the new webxdc info messages. Does `getMessageNotificationInfo` not work for such messages?

Reply:
Hmm, that's true, so we have to call getMessageNotificationInfo in any case, at least just to get the chatName...